### ISSUE TITLE
File transfer issue (again)

### ISSUE THREAD

**Issue by Soulsuke** [emesene/emesene#830]
I admit I don't use file transfer a lot, so I don't know for how long this' been going on, but it looks like that I can't accept incoming file transfers. When sending a file, the progress bar (along with the cancel button) appears on the bottom of the conversation window, but when a file is sent to me, only the desktop notification is shown.
I've seen other bug reports like this (#499 and #744 for instance ), but I can confirm this is still happening... Tried it with both WLM>emesene2 and emesene2>emesene2, latest git. My OS is an up-to-date Debian Sid amd64

**Comment by shuhaowu** [comment]
You know, I think we should just start uploading this file to a file host somewhere.. and send the opponent a link, or have a hook to hook into user's Dropbox or something.. File transfer on non MS client never works well.. It even fails between WLM.. with 2 different versions.

**Comment by Soulsuke** [comment]
Honestly... I hardly use file transfer at all. I'm glad that Opera has that awesome Unite function that works as a webserver for sending and receiving files, it goes way faster than normal msn transfer. Uploading the files somewhere would be a solution, but then wouldn't it just leave the file online after it's done? Also it could be used to spam hosting requests... I recall there was a websender plugin for emesene1, maybe something like that could work as a default file sender-receiver... But how would it be able to actually catch the files that another user has sent?

**Comment by shuhaowu** [comment]
We could first implement something like a image sharer.. file sharer is harder as we need a free, direct link host

**Comment by arielj** [comment]
Is this a dub of https://github.com/emesene/emesene/issues/808 ? if so please close this, try latest git and post on the other issue.

**Comment by c10ud** [comment]
well if talking about faster/simpler, i'm pretty sure that

python -m SimpleHTTPServer 

beats em all

as for the issue, let's ref to #808

**Comment by Soulsuke** [comment]
I don't think it's the same issue, I'm not getting the same errors in the output... Actually, I'm not getting any error at all. The issue seem to be happening quite randomly randomly. I've been trying it running two instances of emesene2, at the beginning i thought it was related to:

[10:20:44 ERROR gtkui.Renderers] this nick: '°-°
[$small][C=c10ud]·$4test·0[/C][$/small]' made the parser go crazy, striping. Error: list index out of range

Since with that combination of nick and personal message nothing happened. I tried changing both, and everything worked fine, then I reverted them back and... It was still working fine. I've tried running emesene -v to get some more infos, but then it started happening randomly, but no error messages would be shown... As for the moment, I can't seem to be able to reproduce it, after the first tries it oddly started to work every time... I don't know what to say at this point, although I'd like to try it with someone using WLM...

Everything's been tried with latest git
